guides/common/sass/settings/_defaults.scss

```diff
@@ -0,0 +1,101 @@
+// TIP set a value to null to disable
+@import "normalize";
+$cursor-default-value: auto !default;
+@import "foundation/components/global";
+@import "compass/css3/box-shadow";
+
+// better defaults (TODO maybe do these at the bottom of the theme colors)
```

Comment:
action required? or keep as is for now?

Reply:
same as above, not mine code, it's from ascii-doc team